sui_programmability/examples/fungible_tokens/sources/private_coin.move

```diff
@@ -0,0 +1,315 @@
+            ];
+
+            pc::split_and_transfer<MIZU>(
```

Comment:
To make sure I understand how all of this works: at this point, an off-chain observer knows that the sum of the balances of the old `coin` and the newly created coin transferred to `owner` is 1000, but doesn't know how much each individual coin is worth?

Reply:
Yes, that is correct. Only the sender knows by default.